BHoM_UI/Templates/MethodCaller.cs

```diff
@@ -93,14 +78,8 @@ public override bool SetItem(object method)
             if (Method == null)
                 return false;
 
-            if (Method.ContainsGenericParameters)
-            {
-                if (Method is MethodInfo)
-                {
-                    Type[] types = Method.GetGenericArguments().Select(x => GetConstructedType(x)).ToArray();
-                    Method = ((MethodInfo)Method).MakeGenericMethod(types);
-                }
-            }
+            if (Method is MethodInfo)
+                Method = ((MethodInfo)Method).MakeGeneric();
```

Comment:
Call the new centralised way of constructing specialised methods from generic ones.